sources/kube.go

```diff
@@ -185,9 +186,14 @@ func (self *kubeSource) GetInfo() (api.AggregateData, error) {
 		return api.AggregateData{}, err
 	}
 	glog.V(2).Info("Fetched list of nodes from the master")
+	events, err := self.eventsApi.GetEvents()
+	if err != nil {
+		return api.AggregateData{}, err
```

Comment:
Should we continue and return stats even if we cannot retrieve events?
The existing code also needs to change as well. So its fine to deal with this in a separate PR.